ksqldb-engine/src/main/java/io/confluent/ksql/structured/SchemaKGroupedStream.java

```diff
@@ -85,7 +85,7 @@
     } else {
       keyFormat = SerdeFeaturesFactory.sanitizeKeyFormat(
           this.keyFormat,
-          true
+          schema.key().size() == 1
```

Comment:
This fix is needed, else `sanitizeKeyFormat()` adds in key wrapping after `groupBy()` removes it in the case of multi-column keys.